corehq/motech/finders.py

```diff
@@ -1,7 +1,15 @@
 from datetime import timedelta
+from operator import eq
 
+from Levenshtein._levenshtein import distance
```

Comment:
oof, import from private module is unfortunate

Reply:
Yeah, I'm not delighted with this library. It was for matching names that have been transliterated from a different alphabet, but [soundex](https://pypi.org/project/soundex/) or [Fuzzy](https://pypi.org/project/Fuzzy/) are probably better for that.